test_runner/src/main/kotlin/ftl/filter/TestFilters.kt

```diff
@@ -73,7 +74,9 @@ object TestFilters {
         }
     }
 
-    fun fromTestTargets(targets: List<String>): TestFilter {
+    fun fromTestTargets(testTargets: List<String>, testTargetsForShard: ShardChunks = emptyList()): TestFilter {
+        val targets = testTargets + testTargetsForShard.flatten()
```

Comment:
I think this is not entirely correct. `testTargets` and `testTargetsForShard` should never be defined at the same time therefore, this line may be misleading for the reader. A new function, for `testTargetsForShard` only would be more self-explanatory. (we can reuse logic common logic ofc)
Let me know your opinion

Reply:
Ironically they can be defined at the same time. 
The documentation 
`## Note: If you include the flags environment-variable or test-targets when running test-targets-for-shard, the flags are applied to all the shards you create.`

It implies if i understand it correctly that the flags can be defined together and they effect each other so our logic is to filter based on the `testTargets` i am merely flattening the testTargetsForShard and further filtering. 

Makes sense?

Reply:
Ah.. you are right, I was confused with the message I got from `gcloud`. My comment is not valid anymore. Thanks for explanation 👍